src/pages/AlumniPage.jsx

```diff
@@ -0,0 +1,84 @@
+import PlaceholderPic from '../assets/placeholderPic.png';
+import Gmail from '../assets/Gmail.png';
+import Facebook from '../images/Facebook.png';
+import LinkedIn from '../images/LinkedIn.png';
+import Footer from "../components/Footer";
+
+export function AlumniPage() {
+    return (
+        <div className="flex flex-col items-center justify-center w-full min-h-screen bg-white p-8">
+            <div className="max-w-[1088px] w-full flex flex-col justify-center items-center gap-10">
+                <h1 className="text-black text-center font-inter text-4xl font-medium" style={{ letterSpacing: '8px' }}>
+                    ALUMNI NETWORK
+                </h1>
+                <p className="text-black text-center font-inter text-base leading-7 px-32">
+                    The Networking Directors of FUSION host a variety of programs to connect the alumni network to FUSION’s current active members, 
+                    to give our members the opportunity to connect and learn from our alumni from everything to professional development to general life advice.
+                </p>
+            </div>
+
+            <img src={PlaceholderPic} alt="Placeholder" className="w-full max-w-[1920px] h-auto mt-8"/>
```

Comment:
The image isn't the full width on my screen, i think it could be on yours, for that let's just get rid of the max-w